### PR TITLE
Create onboarding flow

### PR DESCRIPTION
**Pull request by danielVebman**
## Overview

Create the onboarding flow and make some other miscellaneous changes.

## Changes Made

 - Create onboarding flow
 - Update fonts throughout app
 - Simplify `MorePublicationRow` and update to take a callback for `onFollowPublication`
 - Move long lists into `LazyVStack`s
 - Move main page into `MainView` and put top-level presentation logic in `ContentView` that relies on `isFirstLaunch` 

## Test Coverage

Ran on simulator and device.

## Screenshots

Note: The button in FollowingView doesn't highlight in the video because we don't actually make any calls to the backend, etc. This will change as we integrate networking.

<details>
<summary>Screen recording</summary>

ezgif-6-8b38c4f9bf29  
</details>

### CODE REVIEW

File: Volume/Views/MainView/PublicationList.swift [hunk 1]
```diff
@@ -9,11 +9,9 @@
...
 
 struct PublicationList: View {
-    // The publications a user is following
+    /// The publications a user is following
     private var followedPublications: some View {
-        Section {
-            Header(text: "Following")
-                .padding(.bottom, -12)
+        Section(header: Header("Following")) {
             ScrollView(.horizontal, showsIndicators: false) {
```
Comment (camruss) [inline Volume/Views/MainView/PublicationList.swift:15]: We have too much padding beneath this header compared to the Zeplin designs

File: Volume/Views/MainView/PublicationList.swift [hunk 2]
```diff
@@ -27,24 +25,32 @@ struct PublicationList: View {
...
-            Header(text: "More Publications")
-            ForEach(publicationsData) { publication in
-                NavigationLink(destination: PublicationDetail(publication: publication)) {
-                    MorePublicationRow(publication: publication)
-                        .buttonStyle(PlainButtonStyle())
-                        .padding(.trailing)
+        Section(header: Header("More publications")) {
+            LazyVStack {
+                ForEach(0..<15) { i in
+                    let publication = publicationsData[i % publicationsData.count]
```
Comment (camruss) [inline Volume/Views/MainView/PublicationList.swift:33]: Can we put a TODO flag next to this for our (i.e. my) sanity?

File: Volume/Views/MainView/PublicationList.swift [hunk 3]
```diff
@@ -27,24 +25,32 @@ struct PublicationList: View {
...
+        
+    }
+    
     var body: some View {
         NavigationView {
             ScrollView(showsIndicators: false) {
                 followedPublications
+                Spacer()
+                    .frame(height: 16)
                 notFollowedPublications
```
Comment (camruss) [inline Volume/Views/MainView/PublicationList.swift:54]: `Spacer(minlength: 16)` as a one-liner?

File: Volume/Views/Onboarding/FollowView.swift [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    struct FollowView: View {
+        @AppStorage("isFirstLaunch") private var isFirstLaunch = true
+        @State private var contentOffset: CGPoint = .zero
+        @State private var maxContentOffset: CGPoint = .zero
+        @State private var didFollowPublication = false
+        
+        private var publications: [Publication] {
+            var p: [Publication] = []
+            for i in 0..<10 {
+                let src = publicationsData[i % publicationsData.count]
```
Comment (camruss) [inline Volume/Views/Onboarding/FollowView.swift:21]: TODO

File: Volume/Views/Onboarding/WelcomeView.swift [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+                    .scaledToFit()
+                    .frame(width: 36, height: 36)
+                    .foregroundColor(Color.volume.orange)
+                (
+                    Text(textBold)
+                    .font(.begumBold(size: 16))
+                    +
+                    Text(textRegular)
+                    .font(.begumRegular(size: 16))
+                )
```
Comment (camruss) [inline Volume/Views/Onboarding/WelcomeView.swift:77]: Fix indentation

File: Volume/Views/Onboarding/WelcomeView.swift [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+                .frame(width: 222)
+
```
Comment (camruss) [inline Volume/Views/Onboarding/WelcomeView.swift:79]: delete newline

**Review by camruss — APPROVED** [review]
Very clean

File: Volume/Views/MainView/PublicationList.swift (hunk 3 above)
Comment (danielVebman) [reply Volume/Views/MainView/PublicationList.swift:54]: I think `minLength` would specify the minimum height of the spacer, but I want it to be exactly that height.

**Pull request merged**